docs/jade-getting-started.md

```diff
@@ -223,3 +223,61 @@ cd datarepo-helm-definitions/dev/zzz
 helm namespace upgrade zzz-secrets datarepo-helm/create-secret-manager-secret --version=0.0.4 --install --namespace zzz -f zzzSecrets.yaml
 helm namespace upgrade zzz-jade datarepo-helm/datarepo --version=0.1.0 --install --namespace zzz -f zzzDeployment.yaml
 ```
+
+## 9. Continued Postgres Setup
+1. You should have already added the following to your path: `/Applications/Postgres.app/Contents/Versions/latest/bin` to your path. 
+
+2. Create the data repo db and user by running the following while in the `jade-data-repo` directory. More information can be found in [the database readme](https://github.com/DataBiosphere/jade-data-repo/blob/develop/DATABASE.md)
+  ```
+  psql -f db/create-data-repo-db
+  ```
+  You can run the following to make sure they've been added:
+  This will switch you over to running postgres commands: 
+  ```
+  psql
+  ```
+  List out the databases. You should now see "datarepo" and "stairway."
+  ```
+  \l
+  ```
+
+## 10. Running jade-data-repo and jade-data-repo-ui
+1. Build jade-data-repo: Find more detailed instructions on the [main readme](https://github.com/DataBiosphere/jade-data-repo). You can build the repo and then test to make sure it's working as expceted with the following commands.
+* Build jade-data-repo:
+```
+./gradlew bootRun
+```
+* Run linters and unit tests:
+```
+./gradlew check
+```
+* Run connected and integration tests:
+```
+./gradlew testConnected
+```
+2. Set up env variable for the terminal:
+* export VAULT_ADDR=https://clotho.broadinstitute.org:8200
+* export PROXY_URL=https://jade-sh.datarepo-dev.broadinstitute.org
+* If you're not on a broad issued computer: export HOST=localhost
+
+3. Follow [setup instructions](https://github.com/DataBiosphere/jade-data-repo-ui#jade-data-repository-ui) for the jade-data-repo-ui. 
```

Comment:
Perhaps we could create a single section for setting up all the repositories (or add it to the checkout step)?

Reply:
* I moved terraform setup into this section, so all three repo setup steps are in one section. 
* From my understanding, I think we need to keep this repository setup step separate from the checkout section because we need to do those steps in order: 1) Checkout repositories; 2) GCP, helm & postgres setup; and then 3) trying to build the repos (at least the jade-data-repo).